packages/react-composites/src/composites/ChatComposite/file-sharing/FileUpload.ts

```diff
@@ -87,35 +87,100 @@ export interface FileUploadManager {
 }
 
 /**
- * An internal interface used by the Chat Composite to drive the UI for file uploads.
+ * A class containing {@link EventEmitter} object for emitting events and methods for
+ * subscribing and unsubscribing to file upload events.
  * @beta
  */
-export interface ObservableFileUpload extends FileUploadEventEmitter {
+export class FileUploadEventEmitter {
+  _emitter: EventEmitter;
```

Comment:
nit: make private?
```suggestion
  private _emitter: EventEmitter;
```

Reply:
Needs to be used by a child class `FileUpload`